en/core-libraries/inflector.rst

```diff
@@ -72,20 +69,24 @@ to support other languages, you can use :ref:`inflection-configuration` to
 customize the rules used::
 
     // Apples
-    echo Inflector::pluralize('Apple');
+    echo Inflector::pluralize('apple');
```

Comment:
The comment above is now incorrect with this change. The output would be `apples` with a lowercased a.